views/drones/list.hbs

```diff
@@ -1 +1,28 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>List of drones</h2>
+<hr class="mb-6" />
+<div class="row d-flex justify-content-center">
+
+
+    {{#if drones}}
+
+    {{#each drones as |drone| }}
+
+    <div class="card text-center shadow mb-3 bg-white rounded m-4">
```

Comment:
los valores  {{this.name}} dentro de un each se accede mejor si has hecho    {{#each drones as |drone| }} con {{ drone.name }}, no es que sea más efectivo pero creo que al que lee el template después  le resultará más fácil entender que es ese this.name.

<h3 class="card-title">{{this.name}} </h5> // Se abre h3 y se cierra en h5, un error typo